format/testdata/transform/ifnewlines_before.clj

```diff
@@ -0,0 +1,5 @@
+(if foo? "a"
+  "b")
+
+(if-let [x 123]
+  456 789)
```

Comment:
Should we include negative test cases? For example, to ensure `(if foo? "a" "b")` stays on one line.